ocs_ci/ocs/node.py

```diff
@@ -904,6 +904,53 @@ def delete_and_create_osd_node_vsphere_upi(
     return new_node_name
 
 
+def delete_and_create_osd_node_vsphere_lso(
```

Comment:
Can we also perform ceph side verification post node replacement?
Like replaced host should not list from ceph osd, the rack labels are added correctly etc if any

Reply:
Yes. Do you mean to perform ceph side verification post node replacement with **LSO**?

Reply:
Yes

Reply:
@yitzhak12 is this comment addressed?